interpreter/cling/include/cling/Interpreter/Transaction.h

```diff
@@ -274,15 +274,15 @@ namespace cling {
       return const_reverse_nested_iterator(0);
     }
 
-    void addClangModule(clang::Module* M) {
-      assert(M && "addClangModules: passed clang::Module pointer is null");
+    void addClangDecl(const clang::Decl* D) {
```

Comment:
For us, `Decl`s are always `clang::Decl`s. So please be specific about what kind of `Decl`s this keeps ("deserialized and potentially used"?) rather than providing `Clang` as part of the name.